Dockerfile.windows

```diff
@@ -166,7 +166,7 @@ FROM microsoft/windowsservercore
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 ARG GO_VERSION=1.13.15
-ARG GOTESTSUM_COMMIT=v0.3.5
+ARG GOTESTSUM_COMMIT=v0.5.3
```

Comment:
Was this update needed to make it work?

If not, perhaps better to keep it for a separate PR, and update the linux version to the same version; https://github.com/moby/moby/blob/a70842f9c84f92741877301f33651353eddb44bb/hack/dockerfile/install/gotestsum.installer#L3

(at least a separate commit that updates both would be nice)

Reply:
Now sure about current version. I tried earlier with pkgname-and-test-fails format and that was not available on that old version.

Reply:
OK, if we need the update, probably good to update both the Dockerfile.windows and Dockerfile (linux) here, so that they're on the same version 👍